README.md

```diff
@@ -21,7 +21,7 @@ It eliminates the need to copy, archive or move files.
 
 ## Current status
 
-Currently the plugin is under development. See the links to get more info about the current status.
+Version 1.0.0 has been released!
```

Comment:
Adding a badge seems like a good idea.
It shouldn't be that difficult, I've found some resources on Google, but I think that firstly I have to perform the stable release for that to work
https://badge.fury.io/